micronaut-camunda-bpm-feature/jetty-webapp-and-rest/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/RedirectTest.kt

```diff
@@ -0,0 +1,40 @@
+package info.novatec.micronaut.camunda.bpm.feature.test
+
+import info.novatec.micronaut.camunda.bpm.feature.Configuration
+import io.micronaut.http.HttpRequest
+import io.micronaut.http.client.RxHttpClient
+import io.micronaut.http.client.annotation.Client
+import io.micronaut.http.client.exceptions.HttpClientResponseException
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import io.micronaut.test.support.TestPropertyProvider
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import javax.inject.Inject
+
+@MicronautTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class RedirectTest : TestPropertyProvider {
```

Comment:
rename to "RedirectToWebappsControllerTest" to easily switch between implementation and test